### DISCUSSION TITLE
Do we really need support for IE 11 in v5?

### DISCUSSION THREAD

**Discussion by lorvent** [twbs/bootstrap#30867] [General] [answered]
This is for discussion and not a question to maintainers of bootstrap.

i want to listen both sides of the stories, assuming v5 stable will release in second half of 2020 or early 2021, Do we really need to support IE 11?

Edge beta looks great aswell.

**Comment by mdo** [comment, marked as answer]
We've already dropped IE11 in v5 :).

**Reply by lorvent** [reply]
:) didn't realize that.
